### PR TITLE
* support copying tags

### PR DESCRIPTION
**Pull request by jbostoen**
Tested on iTop 2.7; but not on iTop 3 (I think it will work though).

I added a similar hack to the existing ones to support tags.

Basically the main issue is that while you attempt to copy an object, the "orig_value" correctly gets set. In the GUI, it means tags are visible when creating a copy of an object (before confirmation). However, the elements are therefore not present in the "added" part. Resulting in the lack of tags when objects are created.

This implementation also allows adding/removing tags during the process (before clicking [Create]).

Basically right before submission, the value of the attribute gets rewritten:

* from the original attribute values, all the elements are kept which have not been removed by the user
* a combination of the above + tags added manually by the user is created
* orig_value and removed are both cleared
* added becomes the result of above

### CODE REVIEW

**Comment by Molkobain** [comment]
Hello Jeffrey,

Thanks for the PR but the fix is incorrect, it should be done using a JS hack. The right place for this to work would be in `iTopObjectCopier::CopyAttribute()`.

Do you feel like reworking the PR or should I close it?

**Comment by Molkobain** [comment]
I meant "without" indeed, thanks.
A different version would definitively be considered if it uses the right approach. :)

**Comment by Hipska** [comment]
Hi @jbostoen can you check if a39625cdf282b0a1d2ab0ddfef0732fc436533ee resolves this issue?
